util/goroutine_pool/gp.go

```diff
@@ -101,15 +101,15 @@ func (g *goroutine) put(pool *Pool) {
 }
 
 func (g *goroutine) workLoop(pool *Pool) {
-	timer := time.NewTimer(pool.idleTimeout)
+	timer := time.NewTimer(pool.idleTimeout)  // workLoop函数建议使用time.NewTicker，不必每次都reset，且reset每次都重建一个timer对象，浪费内存
```

Comment:
Please use English to comment the code.

Reply:
Timer is more flexible than ticker, when we have some additional condition, for example:
1. if we get some signal from the channel, don't trigger the timer this round ...
2. otherwise wait the timer to trigger

If you think it's better to change here, please send a PR, rather than comment it in Chinese.